src/main/java/io/strimzi/kafka/bridge/http/HttpBridge.java

```diff
@@ -350,6 +353,21 @@ private void ready(RoutingContext routingContext) {
         HttpUtils.sendResponse(routingContext, httpResponseStatus.code(), null, null);
     }
 
+    private void openapi(RoutingContext routingContext) {
+        FileSystem fileSystem = vertx.fileSystem();
+        fileSystem.readFile("openapiv2.json", readFile -> {
+            if (readFile.succeeded()) {
+                HttpUtils.sendResponse(routingContext, HttpResponseStatus.OK.code(), BridgeContentType.JSON, readFile.result());
```

Comment:
If you are going to access the json as a file (rather than classpath) then it would be simpler and more efficient to use just `HttpServerResponse.sendFile()`.

Reply:
Yeah if we agree to not use classpath (see comment above) then it makes sense.

Reply:
I'm fine with the caching described above, so using `sendFile()` seems like the right way to handle this.